docs/developer/soffits/minimal_soffit.md

```diff
@@ -0,0 +1,71 @@
+
+# Minimal Soffit
+
+A soffit's interaction with the portal is HTTP-based.  It is posible to write a
+soffit in any language or platform that can accept, process, and respond to a
+connection over HTTP.  At the present time, the creators of Soffit expect to
+develop soffits mostly with [Java][] and [Spring Boot][].
+
+## Minimal Soffit Setup Instructions Using Spring Boot
+
+1. Use the [Spring Initializer][] to create a new Spring Boot project with the
+   following settings:
+    * Gradle Project (recommended)
+    * Packaging=*War* (recommended)
+    * Dependencies=*Cache* (recommended) & *Web* (required)
+    * Additional dependencies you intend to use (optional -- you can add them
+      later)
+1. Add Soffit as a dependency to your project (see below)
+1. Add the `tomcat-embed-jasper` dependency to your project (see below)
+1. Add the `@SoffitApplication` annotation to your application class (the one
+   annotated with `@SpringBootApplication`)
+1. Create the directory path `src/main/webapp/WEB-INF/soffit/`
+1. Choose a name for your soffit and create a directory with that name inside
+   `/soffit/` (above);  recommended:  use only lowercase letters and dashes
+   ('-') in the name
+1. Create a `view.jsp` file inside the directory named for your soffit;  add
+   your markup (_e.g._ `<h2>Hello World!</h2>`)
+1. In `src/main/resources/application.properties`, define the `server.port`
+   property and set it to an unused port (like 8090)
+1. Run the command `$gradle assemble` to build your application
```

Comment:
Not at all -- intent was to make it more visually clear that it's a shell command.  Another alternative would be to include a space:  `$ gradle assemble`